docs/docs/started/installing-nebari.md

```diff
@@ -0,0 +1,47 @@
+---
+id: installing-nebari
+---
```

Comment:
```suggestion
title: Installing Nebari
description: A guide to help you install Nebari for your project.
---
```

Reply:
Suggestion - since Nebari is not installed on a per-project basis:

`A guide to help you install Nebari for your team.`